tox.ini

```diff
@@ -56,6 +56,8 @@ markers =
     canary_quick
     cron
     local_mode
+    slow_test
+    release
```

Comment:
Whats the impact of this change? just curious

Reply:
fixes some warnings like:
```
PytestUnknownMarkWarning: Unknown pytest.mark.slow_test - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.slow_test
```

Reply:
great thank you!